.github/workflows/continuous-integration.yml

```diff
@@ -56,6 +56,7 @@ jobs:
         run: npm ci
 
       - name: run remark on changed files
+        if: ${{ !contains(github.event.pull_request.body, 'SKIP_REMARK') }}
```

Comment:
mind either using `env` like the other skippable checks or make them also use this `if` thing